### PR TITLE
Add INSTALL instructions

### PR DESCRIPTION
**Pull request by mortberg**
This resolves issue https://github.com/agda/cubical/issues/12

@Alizter and @Saizan: Can you please take a look and let me know if they are correct and sufficiently detailed.

### CODE REVIEW

**Comment by Alizter** [comment]
LGTM

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+> make
+
+This should compile all of the agda/cubical files. To test that it
+works in emacs run
+
+> emacs Cubical/Core/Primitives.agda
+
+and then type C-c C-l. This should now load the file and you can start
+developing your own cubical files.
+
```
Comment (Saizan) [inline INSTALL]:
```suggestion
You can also register cubical as a library to depend on it in your own Agda developments:

https://agda.readthedocs.io/en/latest/tools/package-system.html
```

**Comment by Saizan** [comment]
I made a suggestion about referencing the library system, but I guess it should be copied to the stack instructions too.

Otherwise it looks good.

**Comment by mortberg** [comment]
Thanks Andrea, that's something I had to help someone with as well!

**Comment by ice1000** [comment]
Why it's not a `.md`?

**Pull request merged**